packages/@markuplint/file-resolver/src/ml-file/ml-file.ts

```diff
@@ -131,12 +142,26 @@ async function stat(filePath: string) {
 	}
 }
 
-function pathNormalize(filePath: string) {
+function pathNormalize(filePath: string, relative?: boolean) {
```

Comment:
Don't you need a default value?

```ts
function pathNormalize(filePath: string, relative = false) {
```

Reply:
Right. It is a quirk of mine. I'll fix it.